.travis.yml

```diff
@@ -145,10 +145,7 @@ script:
   # different builds with CMake
   - if [ "$BUILD_MODE" == "cmake" ]; then
 
-      sudo true;
-      $PIP install cmake==3.8.2;
-      PATH=/usr/local/bin:$PATH;
-      export PATH;
+      $PIP install cmake==3.14.4;
```

Comment:
As we discussed last week, we don’t want to bump this to 3.14. Instead, we want to workaround NAME_WLE usage with lower cmake version.

The reason we see the issue on OSX is that the python library name is different on OSX and we end up in if branch where NAME_WLE is used.

Reply:
Right. I wanted to get the cmake selection working. I't is not yet robust. For example, on linux
```
existing cmake version 3.12.4   location /usr/local/cmake-3.12.4/bin/cmake
Collecting cmake==3.14.4
Could not find a version that satisfies the requirement cmake==3.14.4 (from versions: 0.1.0, 0.2.0, 0.4.0, 0.5.0, 0.6.0, 0.7.0, 0.7.1, 0.8.0, 0.9.0, 3.6.3, 3.6.3.post1, 3.7.2, 3.8.2, 3.9.6, 3.10.3, 3.11.0, 3.11.4, 3.11.4.post1, 3.12.0, 3.13.0, 3.13.1, 3.13.2, 3.13.2.post1, 3.13.3, 3.14.3.post1, 3.14.4.post1, 3.15.3.post1, 3.16.3, 3.16.3.post1, 3.16.5, 3.16.6, 3.16.7, 3.16.8, 3.17.0, 3.17.1, 3.17.2, 3.17.3, 3.18.0, 3.18.2, 3.18.2.post1, 3.18.4, 3.18.4.post1)
```
despite the fact that pypi clearly has cmake-3.14.4-py3-none-manylinux2010_x86_64.whl
I think I'll incorporate a test that demands a match between requested cmake version and the result of 'cmake --version'.
But, yes, it would be nice to get back down to the neighborhood of 3.6.